src/pages/Admin/Admin.tsx

```diff
@@ -1,12 +1,11 @@
-import AuthenticationModal from "components/organisms/AuthenticationModal";
```

Comment:
Note: It would be really good to split this file up into 1 file per component, but that would be best done in a separate PR with no other changes (for simplicity of reviewing)